verify_test.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
```

Comment:
does not belong here. You want it in some out of the way directory, and start test using cmake after `verify` script has been configured and installed